### PR TITLE
add logging images

### PR DESCRIPTION
**Pull request by paynejacob**

### CODE REVIEW

File: images-list [hunk 1]
```diff
@@ -34,3 +34,8 @@ us.gcr.io/k8s-artifacts-prod/external-dns/external-dns rancher/kubernetes-extern
 openpolicyagent/gatekeeper rancher/opa-gatekeeper v3.1.0-beta.11
 k8s.gcr.io/k8s-dns-node-cache rancher/k8s-dns-node-cache 1.15.7
 sonobuoy/sonobuoy rancher/sonobuoy-sonobuoy v0.16.3
+banzaicloud/logging-operator rancher/logging-operator 3.4.0
```
Comment (cbron) [inline images-list:37]: We like to prepend the original org to the image name -> `rancher/banzaicloud-logging-operator`
Also, any image that doesn't currently exist in our dockerhub will need to get a repo set up for it, I can give you details. Ideally that would get checked on PR CI checks, made issue for that.

File: images-list [hunk 2]
```diff
@@ -34,3 +34,8 @@ us.gcr.io/k8s-artifacts-prod/external-dns/external-dns rancher/kubernetes-extern
...
+jimmidyson/configmap-reload rancher/configmap-reload v0.2.2
+paynejacob/fluent-bit-out-syslog rancher/fluent-bit-out-syslog latest
+banzaicloud/fluentd rancher/fluentd v1.10.4-alpine-2
```
Comment (cbron) [inline images-list:40]: We already have a fluentD repo but its custom, so this one should also be org-namespaced. It looks like banzai's is different image entirely than the official fluentD ? 
* https://hub.docker.com/_/fluentd?tab=tags
* https://hub.docker.com/r/banzaicloud/fluentd/tags
* https://hub.docker.com/r/rancher/fluentd/tags

File: images-list [hunk 3]
```diff
@@ -34,3 +34,8 @@ us.gcr.io/k8s-artifacts-prod/external-dns/external-dns rancher/kubernetes-extern
 openpolicyagent/gatekeeper rancher/opa-gatekeeper v3.1.0-beta.11
 k8s.gcr.io/k8s-dns-node-cache rancher/k8s-dns-node-cache 1.15.7
 sonobuoy/sonobuoy rancher/sonobuoy-sonobuoy v0.16.3
+banzaicloud/logging-operator rancher/logging-operator 3.4.0
+jimmidyson/configmap-reload rancher/configmap-reload v0.2.2
+paynejacob/fluent-bit-out-syslog rancher/fluent-bit-out-syslog latest
```
Comment (cbron) [inline images-list:39]: If this image doesn't actually exist anywhere, then lets put version at 0.1.0

File: images-list (hunk 2 above)
Comment (paynejacob) [reply images-list:40]: They may be compatible but this image has all the plugins installed that the operator supports so I think it is worth having 2.

File: images-list (hunk 3 above)
Comment (paynejacob) [reply images-list]: Changed to 0.1.0, this image can be dropped once we have our own ci but until that is ready we can mirror my build.

**Comment by paynejacob** [comment]
https://github.com/rancherlabs/eio/issues/200

**Pull request merged**